src/timesync/timesyncd.c

```diff
@@ -132,6 +132,12 @@ int main(int argc, char *argv[]) {
         if (r < 0)
                 log_warning_errno(r, "Failed to parse configuration file: %m");
 
+        r = manager_parse_fallback_string(m, NTP_SERVERS);
+        if (r < 0) {
+                log_error_errno(r, "Failed to parse fallback server configuration: %m");
+                goto finish;
+        }
```

Comment:
this may fail for OOM too, in theory. I think we really should print a proper message here, so that the error condition is clearly logged.